### ISSUE TITLE
NN Model Visualizations

### ISSUE THREAD

**Issue by woozyking** [EQWorks/snoke-builder-viz#23]
various NN models and their visualizations.

[reference implementation](https://playground.tensorflow.org/#activation=tanh&batchSize=10&dataset=circle&regDataset=reg-plane&learningRate=0.03&regularizationRate=0&noise=0&networkShape=6,2&seed=0.88385&showTestData=false&discretize=false&percTrainData=50&x=true&y=true&xTimesY=false&xSquared=false&ySquared=false&cosX=false&sinX=false&cosY=false&sinY=false&collectStats=false&problem=classification&initZero=false&hideText=false)

**Comment by woozyking** [comment]
@wenting-pan is ffnn the only one to be considered. i remember there were a couple more

> Allow clients to create a neural network model by drag and drop components (enrichable data, network components etc.)
> FFNN
> RNN
> CNN

**Comment by woozyking** [comment]
from slack chat:

> there are a few existing open-source libs that we may either use as-is or build on top of:
> - https://github.com/lutzroeder/Netron - this one visualizes various built models (TensorFlow, PyTorch and many more) but in a rather simple workflow-intended manner (example)
> - https://github.com/tensorspace-team/tensorspace - this one is more interactive and visual. It's similar to ^ that it loads an already trained model and demonstrate its usage and interactivity
> - And ofc the playground.tensorflow.org which is more abstract
